actions/sync/entrypoint

```diff
@@ -37,11 +38,12 @@ sync_dirs() {
   set -x
   cd "${config_repo}${config_path}"
     for dir in $(ls -Ap . | grep "/"); do
-      if [[ -f "${current_repo}/${dir}.syncignore" ]]; then
-        rsync -r "${dir}" "${current_repo}/${dir}" --delete \
-          --exclude=".syncignore" --exclude-from="${current_repo}/${dir}.syncignore"
+      local exclude_list=""
+      if [[ -f "${current_repo}${current_path}/${dir}.syncignore" ]]; then
+        rsync -r "${dir}" "${current_repo}${current_path}/${dir}" --delete \
+          --exclude=".syncignore" --exclude-from="${current_repo}${current_path}/${dir}.syncignore"
```

Comment:
You should always include a `/` between path segments. Extra `/` characters back-to-back do not hurt, but missing `/` characters means that your script breaks.

```suggestion
      if [[ -f "${current_repo}/${current_path}/${dir}.syncignore" ]]; then
        rsync -r "${dir}" "${current_repo}/${current_path}/${dir}" --delete \
          --exclude=".syncignore" --exclude-from="${current_repo}/${current_path}/${dir}.syncignore"
```